src/Hl7.Fhir.Support.Poco/ElementModel/TypedSerialization.cs

```diff
@@ -10,53 +10,28 @@
 using Hl7.Fhir.Model;
 using Hl7.Fhir.Serialization;
 using System;
-using System.Collections.Concurrent;
 using System.Reflection;
 
 namespace Hl7.Fhir.ElementModel
 {
     public static class TypedSerialization
     {
-        private static readonly ConcurrentDictionary<string, ModelInspector> _inspectedAssemblies = new();
-
-        public static ModelInspector GetInspectorForAssembly(Assembly a)
```

Comment:
Do we have here a breaking change? Or was this method not release yet? I am not sure anymore..

Reply:
We haven't released this code yet. It's just 5 days old!

Reply:
Then we can pull it. Thanks